app/config/SessionCacheWiring.scala

```diff
@@ -14,10 +14,26 @@
  * limitations under the License.
  */
 
-package config
-
-import uk.gov.hmrc.http.cache.client.SessionCache
-
-trait SessionCacheWiring {
-  def sessionCache: SessionCache = GmpSessionCache
-}
+///*
+// * Copyright 2019 HM Revenue & Customs
+// *
+// * Licensed under the Apache License, Version 2.0 (the "License");
+// * you may not use this file except in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// *     http://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+// */
+//
+//package config
+//
+//import uk.gov.hmrc.http.cache.client.SessionCache
+//
+//trait SessionCacheWiring {
```

Comment:
If the code in the comments are not needed, can you remove?

Reply:
Done